### PR TITLE
SAK-29813 - New User Webservices

### PR DESCRIPTION
**Pull request by ddelblanco**
New version of these web services following all the comments.

Finally no way to make a SelectionType.ANY

### CODE REVIEW

File: webservices/cxf/src/java/org/sakaiproject/webservices/SakaiScript.java [hunk 1]
```diff
@@ -4392,4 +4392,84 @@ protected Map getServersByServerId(Map servers) {
...
+     *
+     * @param sessionid the id of a valid session
+     * @param userid    the internal user id
+     * @return true/false
+     */
+    @WebMethod
+    @Path("/checkForUserById")
+    @Produces("text/plain")
+    @GET
+    public boolean checkForUserById(
```
Comment (steveswinsburg) [inline webservices/cxf/src/java/org/sakaiproject/webservices/SakaiScript.java]: Should this be admin restricted? Could it be used to fish for valid UUIDs?

File: webservices/cxf/src/java/org/sakaiproject/webservices/SakaiScript.java [hunk 2]
```diff
@@ -4392,4 +4392,84 @@ protected Map getServersByServerId(Map servers) {
...
+    @Path("/isSuperUser")
+    @Produces("text/plain")
+    @GET
+    public boolean isSuperUser(
+            @WebParam(name = "sessionid", partName = "sessionid") @QueryParam("sessionid") String sessionid) {
+        Session s = establishSession(sessionid);
+
+        try {
+            return (securityService.isSuperUser(s.getUserId()));
+        } catch (Exception e) {
```
Comment (steveswinsburg) [inline webservices/cxf/src/java/org/sakaiproject/webservices/SakaiScript.java]: Just noting here that this call doesn't throw an exception itself, it just returns true or false.

Comment (ddelblanco) [inline webservices/cxf/src/java/org/sakaiproject/webservices/SakaiScript.java]: Solved

File: webservices/cxf/src/java/org/sakaiproject/webservices/SakaiScript.java (hunk 1 above)
Comment (ddelblanco) [inline webservices/cxf/src/java/org/sakaiproject/webservices/SakaiScript.java]: I've added the admin restriction. Maybe this was not a very dangerous one, but really I think not admin user won't need this webservice.

**Comment by ddelblanco** [comment]
Modified based in the last comments.

**Pull request merged**